lib/mongodb.js

```diff
@@ -797,6 +797,33 @@ function hasNearFilter(where) {
   return false;
 }
 
+MongoDB.prototype.buildLookup = function (relation, include, model) {
+  var self = this;
```

Comment:
@jannyHou one last question. How to pass options via an api request via http to :
`http://localhost:3000/api/Posts` for example.
I've tested adding it via model's json and it's not working and tried searching but I couldn't find anything.